metricbeat/docs/modules/golang.asciidoc

```diff
@@ -20,7 +20,9 @@ in <<configuration-metricbeat>>. Here is an example configuration:
 ----
 metricbeat.modules:
 - module: golang
-  metricsets: ["expvar","heap"]
+#  metricsets:
```

Comment:
Nit: Could you put the # directly before the `metricset` without space? It's more similar to what we do everywhere else.